src/test/common/test_mutex.cc

```diff
@@ -41,12 +28,6 @@ static void disable_lockdep() {
   }
 }
 
-TEST(Mutex, NormalAsserts) {
```

Comment:
We can keep this test by making the following change:
```cpp
+  testing::FLAGS_gtest_death_test_style = "threadsafe";
   Mutex* m = new Mutex("Normal",false);
   m->Lock();
-  EXPECT_THROW(m->Lock(), int);
+  EXPECT_DEATH(m->Lock(), ".*");
```

Reply:
your fix hangs.

Reply:
@yaozongyou fixed and repushed.